### ISSUE TITLE
Pedestrian multipolygon not rendered if area=yes is missing

### ISSUE THREAD

**Issue by dvdmrtnz** [organicmaps/organicmaps#2393]
If a multipolygon is tagged as `highway=pedestrian` and `area=yes`, it gets rendered correctly:

https://www.openstreetmap.org/relation/1405820#map=18/42.80863/-1.66987

https://omaps.app/8n_T4XVT7d

![](https://user-images.githubusercontent.com/47610359/163355874-b50ff30c-d573-4043-87dc-8d4e8eb1e240.jpg)
![](https://user-images.githubusercontent.com/47610359/163355885-d28577ad-28d9-48d6-a0c1-0cd300e75d63.jpg)

However, if the multipolygon is tagged as `highway=pedestrian` without the `area=yes` tagging, it doesn’t get rendered at all:

https://www.openstreetmap.org/relation/13847953#map=19/42.81189/-1.65802

https://omaps.app/4n_T5DiNfz

![](https://user-images.githubusercontent.com/47610359/163355820-a90aff63-dfb5-4098-ae9f-3ef99abc9cb6.jpg)
![](https://user-images.githubusercontent.com/47610359/163355858-6fbb81d5-25b4-46c2-b4c5-0c73ba1c7dfd.jpg)

This seems like a bug. The wiki says that `area=yes` is not needed for multipolygons.

>for relations area is not used - multipolygons are also marking objects as an area, so area=yes on them would be redundant

https://wiki.openstreetmap.org/wiki/Key:area

**Comment by biodranik** [comment]
@vng can you please take a look?

**Comment by biodranik** [comment]
At a minimum, areas should be routable and searchable. Ideally also visible.
What was the reason to create linear features for pedestrian/footway/service ways?

**Comment by vng** [comment]
Because making areas routabe is a complicated problem.

**Comment by pastk** [comment]
A linear pedestrian way around an areal square doesn't look very good, but is needed for the routing indeed.
	 
A related case is a closed way `waterway=dam` which is also represented in OM as both linear and area features:
![2023-06-12-181144_1366x768_scrot](https://github.com/organicmaps/organicmaps/assets/18434508/ed51a609-4441-4af1-85a4-e2b83f30dca7)

It doesn't look good at all (path captions on dam borders). Ideally we would create an areal feature here only. But how to distinguish with the pedestrian area case? A hardcoded exception?
	 
More common cases are like closed way `amenity=school barrier=fence`. We create an areal schoolgrounds and a linear fence - sounds fine.

And is it possible there is a `waterway=dam barrier=wall`?

**Comment by pastk** [comment]
> Also linear Features are created for area pedestrian/footway/service OSM ways now, while for relations are not. 

An example of multipolygon highway-pedestrian-area, for which OM doesn't create a linear feature:
https://www.openstreetmap.org/relation/10950251#map=18/48.85260/2.36877
![2023-06-14-192129_1366x768_scrot](https://github.com/organicmaps/organicmaps/assets/18434508/eb0d6546-680c-42cc-8ab2-aae7a6650c1f)

Surrounding areas has a linear feature hence there is a dashed border line.

**Comment by pastk** [comment]
@vng did 6c199705a6a12b7e8e0f034f8e21736367f11b03 fix it?

**Comment by dvdmrtnz** [comment]
I think this has been fixed. Tagging in the places shown in the original post has changed since then, but checking other `highway=pedestrian` multipolygons tagged without `area=yes` they seem to be rendering correctly in Organic Maps version ⁦2023.12.18-26⁩:


https://omaps.app/8n_T5Jjh4N/Plaza_del_Juez_Elío
om://8n_T5Jjh4N/Plaza_del_Juez_Elío
https://www.openstreetmap.org/relation/1727158#map=19/42.81759/-1.65730

<img src="https://github.com/organicmaps/organicmaps/assets/47610359/fb46aa6a-2b41-41ca-bb6a-4b637d0af493" width="320"/>

--

https://omaps.app/4n_Tz7g1Al/Paseo_Santxiki
om://4n_Tz7g1Al/Paseo_Santxiki
https://www.openstreetmap.org/relation/1364762#map=17/42.80023/-1.61372

<img src="https://github.com/organicmaps/organicmaps/assets/47610359/d3d89a25-2e4a-4ef1-9dd6-c19bc6dbcf18" width="320"/>

**Comment by pastk** [comment]
Yes the generator now adds `area=yes` to multipolygon relations explicitly.

But why those areas don't have border lines then?

**Comment by dvdmrtnz** [comment]
> But why those areas don't have border lines then?

I don’t know why, but I think they look better without an outline

**Comment by dvdmrtnz** [comment]
Related to https://github.com/organicmaps/organicmaps/issues/7075

**Comment by vng** [comment]
This is how it looks now with bicycle layer enabled :)

<img width="2768" height="1594" alt="Image" src="https://github.com/user-attachments/assets/b9f26551-986c-4541-87e1-4363a5e2dbe5" />
